.travis.yml

```diff
@@ -19,6 +19,22 @@ matrix:
       env:
         - OS_TYPE=doxygen
       addons: { apt: { packages: ["cmake", "graphviz"] } }
+    - compiler: "gcc"
+      os: osx
+      osx_image: xcode9.2
+      env:
+        - EXTRA_SSH=-oHostKeyAlgorithms=+ssh-dss
```

Comment:
curios: are we planning to store stuff on a server?
this env-var `EXTRA_SSH` is used in the upload scripts;
depending on how/if this is done, this is needed or not;

also, I would drop xcode9.2: it's really old;
xcode10 is old-ish;

at the rate Apple is updating Xcode, we might see an Xcode 12 pretty soon

Reply:
We'll have to store the artifacts somewhere. If it's not a problem, we can leave the "extra_ssh" there for now.

Reply:
Fine to leave it